core/game.py

```diff
@@ -90,15 +92,6 @@ def remove_user_by_name(self, name: str):
             User(name=name)
         )  # this works because users are hashed by user.name
 
-    def start_polling_phase(self, poll: Poll):
-        self.user_data = {
-            k: {user: None for user in self.user_data if user is not k}
-            for k, v in self.user_data.items()
-        }
-        self.phase = Phase.POLLING
-        self.poll = poll
```

Comment:
Tu się ustawiało self.poll, dlaczego z tego zrezygnowaliśmy?

Reply:
imo, powinien być na to specjalny endpoint, a jeżeli nie, to lepiej byłoby to zrobić w start_game. Tam też przerzuciłem resztę logiki z start_polling_phase

Reply:
dobra, dodałem